src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -141,13 +141,12 @@ export function PreSubmitSection(props) {
             {statementOfTruthBodyElement(form?.data, statementOfTruth.body)}
             <p>
               I have read and accept the{' '}
-              <a href="/privacy-policy/" target="_blank">
+              <a
+                href="/privacy-policy/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
                 privacy policy
-                <i
```

Comment:
we don't want this? I thought this was common for external links

Reply:
No, we don't.  This is the Design-System guidance page I found: 
https://design.va.gov/content-style-guide/links#linking-to-external-sites \[see the **Linking to external sites** section > bullet# 4\].

Reply:
The guidance also says though 'Within components, for brevity, use of the icon is acceptable.' which this qualifies for. I'm sorry I missed this before.